frontend/src/bundles/common/hooks/hooks.ts

```diff
@@ -3,4 +3,4 @@ export { useAppForm } from './use-app-form/use-app-form.hook.js';
 export { useAppSelector } from './use-app-selector/use-app-selector.hook.js';
 export { useField as useFormField } from 'formik';
 export { useCallback, useEffect, useMemo, useState } from 'react';
-export { useLocation } from 'react-router-dom';
+export { Navigate, useLocation } from 'react-router-dom';
```

Comment:
My bad, Navigate is not a hook, but looks more like a component, maybe you can import it from there

Reply:
this is root file for hooks